examples/with-react-md/static/react-md.light_blue-yellow.min.css

```diff
@@ -0,0 +1 @@
+../node_modules/react-md/dist/react-md.light_blue-yellow.min.css
```

Comment:
Correct. Or you can just copy the file. It would be better to somehow integrate CSS and fonts bundling with Webpack, but I have no idea on how to implement this at the moment.